### DISCUSSION TITLE
More output at result?

### DISCUSSION THREAD

**Discussion by ffgg8686** [neonbjb/tortoise-tts#185] [Q&A]
Hi, I'm really enjoying this script - thank you for making this available. I was wondering - every time I generate voice, it only produces 3 per each time. I find there are few snippets here and there that are useful - having more output will be able to make even better combination. Is something like that a possibility? Thanks.

**Comment by neonbjb** [comment]
Yes, it's an api option. You'll need to make some slight tweaks to the code.
